Organic/Affiliate.php

```diff
@@ -12,6 +12,7 @@ public function __construct( Organic $organic ) {
         $this->organic = $organic;
         add_action( 'init', [ $this, 'register_gutenberg_block' ] );
         add_action( 'admin_enqueue_scripts', [ $this, 'register_scripts' ] );
+        add_action( 'admin_enqueue_scripts', [ $this, 'admin_post_page_enqueue' ] );
```

Comment:
I'd prefer to enqueue all the scripts in the single `register_scripts` method rather that having multiple hooks

Reply:
Also, I think it would be nice to wrap admin-related stuff/hook registration into `is_admin()` condition.